### PR TITLE
Storybook version update and other dependencies

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves N/A

**Overall change:**
Use version 5.1.3 of Storybook. This is version contains a fix for a bug when viewing components in a new tab canvas. More info here: https://github.com/storybookjs/storybook/issues/6800

## Updated:

+ prettier@1.18.2
+ lerna@3.15.0
+ eslint-plugin-import@2.17.3
+ @storybook/addon-notes@5.1.3
+ stylelint-processor-styled-components@1.8.0
+ @storybook/theming@5.1.3
+ @storybook/addon-viewport@5.1.3
+ styled-components@4.3.1
+ @storybook/addon-actions@5.1.3
+ @storybook/addon-a11y@5.1.3
+ @storybook/addon-knobs@5.1.3

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~~Tests added for new features~~
- [ ] Test engineer approval

### CODE REVIEW

**Review by DenisHdz — CHANGES_REQUESTED** [review]
CHANGELOG needs to be updated :)

**Comment by j-pendlebury** [comment]
@DenisBBC I didn't bump the version of Psammead, which means no need for CHANGELOG update. Unless I need to bump version of Psammead as well? 🤔

**Comment by j-pendlebury** [comment]
@DenisBBC done now 😄

**Review by DenisHdz — DISMISSED** [review]

**Review by dr3 — DISMISSED** [review]

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
There are a bunch of other storybook updates available, e.g. https://github.com/bbc/psammead/pull/603 Suggest they are all done at once.

File: CHANGELOG.md
Comment (dr3) [inline CHANGELOG.md]:
```suggestion
| 1.1.8   | [PR#594](https://github.com/BBC/psammead/pull/594) Bump various dependencies inc storybook and lerna |
```

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by DenisHdz — APPROVED** [review]

File: packages/utilities/psammead-assets/src/svgs.test.jsx
Comment (andrew-nowak) [inline packages/utilities/psammead-assets/src/svgs.test.jsx:12]: 🤩

**Review by andrew-nowak — APPROVED** [review]
This introduces a warning in the console in storybook:
`The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".`
But this is due to a known issue in storybook: https://github.com/storybookjs/storybook/issues/6998 and I don't think blocks this going forward :)

Otherwise lgtm but needs code owner approval

**Review by dr3 — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]

**Comment by j-pendlebury** [comment]
After updating versions of all the dependencies in this PR, the inputProvider doesn't update values for different stories, e.g.
image
The SVG width doesn't update from the news one, which makes the SVG look tiny

image
Headline text doesn't update to Afrique, stays English instead.

Will raise a new PR to update to version 5.1.4 of Storybook, which includes some `core-js` bug fixes.

**Pull request closed without merging**
